bin/adb-reconnect.sh

```diff
@@ -2,11 +2,5 @@
 
 while read d
 do
-  if [ -z "$STF_PROVIDER_PUBLIC_IP" ]; then
```

Comment:
keep in mind that despite the comment below, STF_PROVIDER_PUBLIC_IP variable is not a marker of "real devices".